commercetools-internal-docs/src/main/java/io/sphere/sdk/meta/ReleaseNotes.java

```diff
@@ -134,22 +131,6 @@
  <li class=fixed-in-release></li>
  </ul>
  -->
- <h3 class=released-version id="v1_21_0">1.21.0 (18.07.2017)</h3>
```

Comment:
This looks like you accidentially removed our release notes, should be fixed by merging master to this branch.